build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -1510,6 +1510,12 @@ message BatchUpdateBlobsRequest {
 
     // The raw binary data.
     bytes data = 2;
+
+    // The format of `data`. Must be `IDENTITY` (or unspecified), or one of the
```

Comment:
Nit: ProtoV3 doesn't differentiate between `IDENTITY` and unspecified. That '(or unspecified)' part could be dropped.

Reply:
It could, but I don't think it hurts to mention this.

Reply:
Maybe remove the word 'or' then?

> Must be `IDENTITY` (unspecified), or [...]

Reply:
How about `IDENTITY`/unspecified ?

Reply:
Sure! As long as we don't phrase it in a way that could be interpreted as being two different representations.

Reply:
Done.